units/units.cpp

```diff
@@ -144,12 +144,14 @@ static const umap base_unit_names{
                                        // 1000 m^3 not mega meters cubed
     {kg, "kg"},
     {mol, "mol"},
+    {unit(0.1, m), "dm"},  // don't want to use deci in most conversion contexts
+                           // but for meter it is fine
     {A, "A"},
     {A * h, "Ah"},
     {V, "V"},
     {s, "s"},
     // this is so Gs doesn't get used which can cause issues
-    {giga * s, "Bs"},
+    {giga * s, "Bs"},  // B is for billion
```

Comment:
Is this comment (regarding `Gs`) still true with domains?

Reply:
`Gs` is a symbol for Gauss so the domains haven't affected that